Tests/Admin/AdminTest.php

```diff
@@ -22,7 +22,7 @@ class AdminTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->contextManager = $this->getMock('Sonata\ClassificationBundle\Model\ContextManagerInterface');
+        $this->contextManager = $this->getMockForAbstractClass('Sonata\ClassificationBundle\Model\ContextManagerInterface');
```

Comment:
Not sure about this, IMO is better to do like this: https://github.com/sonata-project/SonataUserBundle/pull/840

Follows the styles of how we did on other bundles.